main.tf

```
@@ -21,14 +21,17 @@ const arrayRes = JSON.parse(docsStr).reduce((acc, doc, ind) => {
     doc.pages.forEach((page) => {
       if (page && page.inputs) {
         Object.values(page.inputs).forEach((input) => {
-          if (input && ((input.reportingName !== '' && input.reportingName) && input.value)) {
-            if (input.type === 'radio' && input.options) {
+          if (input && ((input.reportingName !== '' && input.reportingName) || input.type === 'radio' && input.value)) {
```

We might want to have the or condition also wrapped in parens for easier reading:
```suggestion
          if (input && ((input.reportingName !== '' && input.reportingName) || (input.type === 'radio' && input.value))) {
```